UnchainedLauncherCore/src/API/RCON.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnchainedLauncher.Core.API
+{
+    public class RCON
+    {
+        public static async Task sendCommand(IPEndPoint rconLocation, string command)
```

Comment:
I think RCON should be a class which holds on to an IPEndPoint, and then has a `SendCommand(string command)` method.  This way RCON can also manage the state of a connection and hold it open without leaking any of that abstraction out

Reply:
on the plugin-side, the server kills the connection once it sees a newline character so there's no way to send multiple commands per connection. It's such a low-frequency event that I think the overhead of one command per connection is acceptable. This was done initially so that RCON commands can easily be sent via netcat and I'd like to maintain that for hassle-free debugging if it's needed.